parse/table_definitions_bsc/pancakeswap/UniswapV3Pool_event_Burn.json

```diff
@@ -0,0 +1,87 @@
+{
+    "parser": {
+        "type": "log",
+        "contract_address": "SELECT pool FROM ref('UniswapV3Factory_event_PoolCreated')",
```

Comment:
Can you create a separate PR for tables that reference `UniswapV3Factory_event_PoolCreated`? The `UniswapV3Factory_event_PoolCreated` will need to be created first in order for the rest of the tables to be created.

Reply:
Split PR is [here](https://github.com/nansen-ai/evmchain-etl-table-definitions/pull/139).